### PR TITLE
automate locking closed threads

### PR DESCRIPTION
**Pull request by techknowlogick**
With comments happening on closed issues/prs this locks issues that have been closed for >45days. This allows for comments on recently closed issues/prs to still happen.

### CODE REVIEW

File: .github/workflows/lock.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+concurrency:
+  group: lock
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v4
+        with:
+          issue-inactive-days: '45'
```
Comment (silverwind) [inline .github/workflows/lock.yml]:
```suggestion
          issue-inactive-days: 45
```

Will work as number and is actually documented as such in the parameters list.

File: .github/workflows/lock.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    # Run once a day
+    - cron: '0 0 * * *'
```
Comment (silverwind) [inline .github/workflows/lock.yml]:
```suggestion
    - cron: '0 0 * * *' # Run once a day
```

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
Should `.github/lock.yml` be deleted here?

**Comment by techknowlogick** [comment]
> Should `.github/lock.yml` be deleted here?

@silverwind deleted, thanks for catching that :)

**Review by lunny — APPROVED** [review]

**Pull request merged**